src/docs/product/performance/retention-priorities/index.mdx

```diff
@@ -0,0 +1,127 @@
+
+Some projects within your organization are very high volume compared to others. This means that low-volume projects might get drowned out if we treat all projects equally. To ensure that you have enough samples regardless of project scale, we prioritize low-volume projects.
+
+#### Low Volume Transactions
+
+Transactions can be logically grouped by their name within a project. This means that transactions with the same name most likely indicate the same operation or code path, while transactions with different names usually indicate different operations.
+We prioritize low volume transactions for the same reason we prioritize low volume projects - so they don't get drowned out by high volume transactions. We want to make sure that transactions representing low volume code paths are sampled in enough quantities to generate a complete view of your application.
+
+### Deprioritized Transactions
+
+Sentry automatically deprioritizes the following transactions:
+
+#### Health Checks
+
+We deprioritize health check type transactions because while they're important for checking the stability of your application, they don't have any value beyond the task associated with them. We consider a transaction to be a health check if its name matches one of the following glob patterns:
+
+- `*healthcheck*`
+- `*healthy*`
+- `*live*`
+- `*ready*`
+- `*heartbeat*`
+- `*/health`
+- `*/healthz`
+- `*/ping`
+
+### How to Change Retention Priorities
+
+To make changes to a project's retention priorities in Sentry, go to **Settings > Projects**, select the project you're interested in, then scroll down to "Performance" in the "PROCESSING" section and make any updates under "RETENTION PRIORITIES" using the toggles. Each of your projects can have an individual set of retention priorities.
+
+<div style="position: relative; padding-bottom: calc(52.60977630488816% + 41px); height: 0; width: 100%">
+  <iframe
+    src="https://demo.arcade.software/ETgxCQqyWN5uuCityFxT?embed"
+    frameborder="0"
+    loading="lazy"
+    webkitallowfullscreen
+    mozallowfullscreen
+    allowfullscreen
+    style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;color-scheme: light;"
+    title="General Settings — totes-ma-gotes — Sentry"
+  ></iframe>
+</div>
+
+## Deciding on Your SDK Sample Rate
+
+To give Sentry the fullest, most accurate picture of your application's health, we recommend that you send us 100% of your transactions. But if sending all your transactions doesn't fit your needs, we offer two SDK sampling configuration options:
+
+The `tracesSampleRate` option enables you to control the percentage of transactions you send to Sentry.
+
+The `TracesSampler` function (where available), allows you to manually target the specific transactions you send to Sentry and controls the sample rate based on the transaction itself and the context in which it's captured.
+
+The two options are mutually exclusive. If you set up both, `tracesSampler` will take precedence.
```

Comment:
I would delete then this whole section.

Reply:
If we link to tracesSampler and disclose the metering policy in the above suggestion we should remove this block.